plugins/hubspot/src/App.tsx

```diff
@@ -102,6 +102,8 @@ export function App() {
 
     const [, navigate] = useLocation()
 
+    const isAuthenticated = auth.isAuthenticated()
```

Comment:
It's used in the `useEffect` hook so that if a user authenticates via the CMS we need to fetch all their plugin contexts.

Reply:
Hmm. I guess I understand how that `useEffect` ends up being re-evaluated, because `useLocation` hook above likely fires when something authentication-related happens, and `isAuthenticated` gets updated too. But then probably best not do `auth.isAuthenticated()` again inside `getContexts`?